tests/end-to-end/upgrade/pkg/tests/event-bus/event_bus.go

```diff
@@ -0,0 +1,399 @@
+package eventbus
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httputil"
+	"strings"
+	"time"
+
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	api "github.com/kyma-project/kyma/components/event-bus/api/publish"
+	subApis "github.com/kyma-project/kyma/components/event-bus/api/push/eventing.kyma-project.io/v1alpha1"
+	eaClientSet "github.com/kyma-project/kyma/components/event-bus/generated/ea/clientset/versioned"
+	subscriptionClientSet "github.com/kyma-project/kyma/components/event-bus/generated/push/clientset/versioned"
+	"github.com/kyma-project/kyma/components/event-bus/test/util"
+	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	eventType           = "test-e2e"
+	subscriptionName    = "test-sub"
+	eventActivationName = "test-ea"
+	srcID               = "test.local"
+
+	success     = 0
+	fail        = 1
```

Comment:
success/fail constants are not used? Should them be here?

Reply:
I removed them.